recipes-kernel/linux/linux-nilrt-next_git.bb

```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "NILRT linux kernel next development build"
+
+require linux-nilrt-next.inc
+
+LINUX_VERSION_EXTENSION = "-next"
+KERNEL_PACKAGE_NAME = "kernel${LINUX_VERSION_EXTENSION}"
```

Comment:
I could make a weak argument that it would be good to know from the files on disk and the bzImage symlink if you are running a 'next' kernel but yeah, it's a bit redundant so I'll take it out.